google/gax/grpc.py

```diff
@@ -30,8 +30,9 @@
 """Adapts the grpc surface."""
 
 from __future__ import absolute_import
+
 from grpc import RpcError, StatusCode
-from . import auth
+from . import _grpc_oauth2client
```

Comment:
Yeah, fixed. I also did away with the use of closures and went with @dhermes' suggestion of using a class like we do in google-auth.